### ISSUE TITLE
Include MDN links in property IntelliSense

### ISSUE THREAD

**Issue by connorshea** [microsoft/vscode-css-languageservice#150]
I'm sure I've mentioned this before or discussed it in another issue, but I couldn't find anything specifically about it in the existing issues.

I'd like to be able to hover over, e.g. `background-color` and - besides the description that's already there - have a clickable link to [the MDN documentation](https://developer.mozilla.org/docs/Web/CSS/background-color) for the property.

What it looks like right now:

<img width="829" alt="Screen Shot 2019-04-02 at 12 17 09 PM" src="https://user-images.githubusercontent.com/2977353/55426138-488dd280-5541-11e9-9497-afc0252618f8.png">

The browser-compat-data package includes `mdn_url`s for most (all?) CSS properties, so it should be possible to derive the documentation links from that package, like is already done for browser support.

https://github.com/mdn/browser-compat-data/blob/3268af01c1bf6059f3d7a25c1c212ac4522c291f/css/properties/background-color.json#L6

There's a similar issue for the HTML service: https://github.com/Microsoft/vscode-html-languageservice/issues/12

**Comment by octref** [comment]
I'll probably do it in May. There is a requirement that if a MS product links to external websites, it needs to get explicit consent from user. That means we can't just embed the link directly.

Also there are work that needs to be done to convert the auto-completion's documentation to use Markdown format.

**Comment by connorshea** [comment]
@octref how would you imagine the consent process to be? A notification the first time they run the extension that would toggle a boolean setting, or would it be a popup that shows itself every time someone clicks the link (I assume it would have an 'Always Allow' option)? And would you accept a PR that introduced that behavior? :)

It wouldn't really serve any purpose without the links/markdown formatting implemented, I just want to help get it going.

**Comment by octref** [comment]
> A notification the first time they run the extension that would toggle a boolean setting

Notification on first time clicking an external link of specific domain, say `https://developer.mozilla.org`. If user accepts, it's added to a setting (or cache?) and wouldn't show notification next time.

> And would you accept a PR that introduced that behavior? 

Not yet, I'll work on this in June. https://github.com/microsoft/vscode-css-languageservice/issues/132 needs to happen first. I also want to make sure for custom loaded data, people can do similar things such as linking custom properties to their doc websites.

**Comment by connorshea** [comment]
🎉
